### PR TITLE
Features/MovieList

### PR DESCRIPTION
**Pull request by paulgrym**
Add a MovieList component with a test fetch from popular movies API.

I've made a MovieList component and to do this I had to change Section component structure and styling. I also changed max-width of the Header component to adjust its position to the Section position. Please check if I have spoiled your ideas for these components a lot :D. 
I did a traditional fetch from API to check the styling with the real data, but I'll work on that more when it comes to using Redux :)
Finally, I moved back google fonts to index.html, because I spotted a warning in the console saying that for today
 it is not recommended.

### CODE REVIEW

File: src/features/movies/MovieList/styled.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 324px);
+  grid-gap: 24px;
+  justify-content: center;
+  margin: 12px auto 40px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}){
+    grid-template-columns: 1fr;
+    margin-bottom: 32px;
+  }
```
Comment (MieczyslawFrontDev) [inline src/features/movies/MovieList/styled.js:13]: There should be grid-gap: 16px as in the project. You can use this property on small breakpoint.

Comment (MieczyslawFrontDev) [inline src/features/movies/MovieList/styled.js:13]: There should be grid-gap: 16px as in the project. You can use this property on small breakpoint.

File: src/features/movies/MovieList/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { MovieTile } from "./MovieTile";
+import { Wrapper } from "./styled";
+import { usePopularMovies } from "./usePopularMovies";
+
+export const MovieList = () => {
+  const popularMovies = usePopularMovies();
+
+  // console.log(popularMovies);
```
Comment (MieczyslawFrontDev) [inline src/features/movies/MovieList/index.js:8]: Unnecessary comment

**Review by MieczyslawFrontDev — APPROVED** [review]
Great work ! I don't have more sugesstions. Everything looks well.

File: src/features/movies/MovieList/styled.js (hunk 1 above)
Comment (paulgrym) [reply src/features/movies/MovieList/styled.js:13]: Good point! I've added it to small screens :D

**Pull request merged**